src/main/java/gregtech/loaders/recipe/ComponentRecipes.java

```diff
@@ -0,0 +1,387 @@
+package gregtech.loaders.recipe;
+
+import gregtech.api.recipes.RecipeMaps;
+import gregtech.api.unification.material.MarkerMaterials;
+import gregtech.api.unification.material.Materials;
+import gregtech.api.unification.ore.OrePrefix;
+import gregtech.common.items.MetaItems;
+import net.minecraftforge.fluids.FluidStack;
+
+import static gregtech.api.GTValues.L;
+
+public class ComponentRecipes {
+
+    private static final FluidStack[] pumpFluids = {Materials.Rubber.getFluid(L), Materials.StyreneButadieneRubber.getFluid((L * 3)/4), Materials.SiliconeRubber.getFluid(L/2) };
+
+    public static void register() {
+
+        //Field Generators start ---------------------------------------------------------------------------------------
```

Comment:
I like start comment telling what is created. Though it should start with `S` to be consistent